packages/typescript-estree/jest.config.js

```diff
@@ -1,8 +1,13 @@
 'use strict';
 // @ts-check
 
+console.log('with', process.env.TYPESCRIPT_ESLINT_PROJECT_SERVICE);
+
 /** @type {import('@jest/types').Config.InitialOptions} */
 module.exports = {
   ...require('../../jest.config.base.js'),
   testRegex: ['./tests/lib/.*\\.test\\.ts$'],
+  testPathIgnorePatterns: process.env.TYPESCRIPT_ESLINT_PROJECT_SERVICE
+    ? ['/node_modules/', 'project-true']
+    : [],
```

Comment:
`parse.project-true.test.ts` never really needed to be tested with the project service on anyway.